src/OSPSuite.Core/Domain/Services/SensitivityAnalyses/SensitivityAnalysisRunner.cs

```diff
@@ -76,7 +82,12 @@ public async Task Run(SensitivityAnalysis sensitivityAnalysis, SensitivityAnalys
 
       public void Stop()
       {
-         _sensitivityAnalysisEngine?.Stop();
+         if (_cancellationTokenSource != null)
+         {
+            _cancellationTokenSource.Cancel();
```

Comment:
shoul'dnt we set _sensitivityAnalysisEngine to null? It was a given before but I am not sure if this is the case now?

Reply:
Doesn't this code trigger the `finally` where the engine is set to null?